packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -166,6 +167,10 @@ const VisitHeader: React.FC = () => {
     });
   }, []);
 
+  const { person } = useOmrsRestPatient(patient?.id);
+  const isDeceased = Boolean(person?.dead);
+  const deceased = (!isDeceased && currentVisit) || (!isDeceased && !currentVisit)
```

Comment:
thanks @jwnasambu, you having an es-lint issue here that might be causing the build to fail. You missed a semicolon and the end `const deceased = (!isDeceased && currentVisit) || (!isDeceased && !currentVisit);`

Reply:
@Jexsie thanks so so much for guide .

Reply:
There is no real reason for the `deceased` variable and it's name is confusing (it's functionally equivalent to:

```ts
const deceased = !isDeceased;
```

I would just use `!isDeceased` in the one check we use it for...